Source/inv.cpp

```diff
@@ -1188,6 +1230,15 @@ void DrawInvBelt(const Surface &out)
 
 		DrawItem(myPlayer.SpdList[i], out, position, sprite);
 
+		if (*sgOptions.Gameplay.enableFloatingInfoBox) {
+			// Draw info box
+			if (pcursinvitem == i + INVITEM_BELT_FIRST) {
+				if (ControlMode == ControlTypes::KeyboardAndMouse || invflag) {

```

Comment:
I think this needs to be fixed or else non K&M users won't be able to have floating info box?